examples/hello_ocean.jl

```diff
@@ -0,0 +1,35 @@
+# # Hello, ocean!
+#
+# > How inappropriate to call this planet Earth, when it is quite clearly _Ocean_.
+#
+#   --Arthur C. Clark
+
+using Oceananigans, Oceananigans.Grids, Plots
+
+grid = RegularCartesianGrid(size = (1, 64, 64),
+                            x = (0, 1), y = (-4, 4), z = (-4, 4),
+                            topology = (Periodic, Bounded, Bounded))
+
+model = IncompressibleModel(grid = grid,
+                            architecture = CPU(),
+                            advection = Oceananigans.Advection.WENO5(),
+                            tracers = :b,
+                            buoyancy = BuoyancyTracer(),
+                            closure = IsotropicDiffusivity(ν=1e-3, κ=1e-3))
+         
+@info "Simulating a rising buoyant bubble with" model
+
+set!(model, b = (x, y, z) -> exp(-y^2 - z^2))
```

Comment:
Is, perhaps, `exp(-(y^2 + z^2))` more soothing for the eye?

Reply:
that's more characters!

Reply:
my question was whether it's more soothing for the eye not whether it's easier for the fingers of the person who types it...

Reply:
characters strain the eye, as well as typing fingers...

Reply:
ok, fair enough.

But it bothers me that y kisses the minus sign before it while there is some distance between - and z... Seems like y and z are not treated equally..